src/privatesend-server.cpp

```diff
@@ -499,9 +499,8 @@ void CPrivateSendServer::CheckTimeout(CConnman& connman)
 
     if(!fMasternodeMode) return;
 
-    int nLagTime = fMasternodeMode ? 0 : 10000; // if we're the client, give the server a few extra seconds before resetting.
     int nTimeout = (nState == POOL_STATE_SIGNING) ? PRIVATESEND_SIGNING_TIMEOUT : PRIVATESEND_QUEUE_TIMEOUT;
-    bool fTimeout = GetTimeMillis() - nTimeLastSuccessfulStep >= nTimeout*1000 + nLagTime;
+    bool fTimeout = GetTime() - nTimeLastSuccessfulStep >= nTimeout;
```

Comment:
Is the removal of `+ nLagTime` also intended here?

Reply:
yes, cause it's always `0` on masternode side :)

see `-      int nLagTime = fMasternodeMode ? 0 : 10000;` above

Reply:
Ah, ok :+1: